lib/routes/packages.js

```diff
@@ -96,13 +88,15 @@ module.exports = function (app) {
    *       $ref: '#/responses/Packages'
    */
   app.routes.get('/packages', auth, function (req, res, next) {
-    app.models.PackageCache.findAll({ partitioner: 'cached' })
-      .once('error', err => {
+    app.models.Package.model.scan({ }, function (err, data = {}) {
```

Comment:
`Package.findAll` will implement the scan here so you don't need to use the underlying model directly

Reply:
Yeah, gotcha, thanks for reminding. Had used it in `bffs` as well.